src/pages/permissions/CreateIdentityModal.tsx

```diff
@@ -64,7 +30,7 @@ const CreateIdentityModal: FC<Props> = ({ onClose }) => {
     <Modal
       close={onClose}
       className="create-tls-identity"
-      title={token ? "Identity has been created" : "Generate trust token"}
+      title={`Identity '${identityName}' created`}
```

Comment:
```suggestion
      title={`Identity ${identityName} created`}
```
We don't put user provided values in quotes, usually. We sometimes use bold text, but maybe here we can inline it plainly.